src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java

```diff
@@ -244,8 +247,8 @@ private void assertExtractedDetailsAreCorrect(List<PayeeACHAccountExtractDetail>
             assertEquals("Wrong payment detail first name", expectedDetail.getFirstName(), actualDetail.getFirstName());
             assertEquals("Wrong payment detail payment type", expectedDetail.getPaymentType(), actualDetail.getPaymentType());
             assertEquals("Wrong payment detail balance account indicator", expectedDetail.getBalanceAccount(), actualDetail.getBalanceAccount());
-            assertEquals("Wrong payment detail completion date", expectedDetail.getCompletedDate(), actualDetail.getCompletedDate());
-            assertEquals("Wrong payment detail bank name", expectedDetail.getBankName(), actualDetail.getBankName());
+//            assertEquals("Wrong payment detail completion date", expectedDetail.getCompletedDate(), actualDetail.getCompletedDate());
+//            assertEquals("Wrong payment detail bank name", expectedDetail.getBankName(), actualDetail.getBankName());

```

Comment:
Remember to remove these commented-out lines if they are truly no longer needed.